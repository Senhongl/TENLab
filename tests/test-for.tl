a = [1, 2, 3];
b = 1;
for (i in a) {
    print(i);
}