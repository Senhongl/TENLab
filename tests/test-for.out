 1
[ CPUIntType{1} ]
 2
[ CPUIntType{1} ]
 3
[ CPUIntType{1} ]
